lib/xgu/xgux.h

```diff
@@ -3,13 +3,71 @@
 
 #include "xgu.h"
 
+#define MIN(a,b) ((a)<(b)?(a):(b))
+#define MAX_BATCH 120
+
 inline
 void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int count) {
     uint32_t *p = pb_begin();
     p = xgu_begin(p, mode);
     p = xgu_draw_arrays(p, start, count);
     p = xgu_end(p);
-    p = pb_end(p);
+    pb_end(p);
+}
+
+inline
+void xgux_draw_elements16(XguPrimitiveType mode, unsigned int count, const uint16_t* elements) {
+    uint32_t *p = pb_begin();
+    p = xgu_begin(p, mode);
```

Comment:
These supposedly have a maximum number of vertices / primitives / ...?
Same issue as for the `pb_begin/end` hack below: might need additional logic.

Needs a comment in XGU branch after merge to remind us.